batch/batch/front_end/front_end.py

```diff
@@ -500,9 +500,11 @@
 ''',
                                           jobs_args)
                 except pymysql.err.IntegrityError as err:
-                    if err.args[1] == 1022:
+                    # 1062 ER_DUP_ENTRY https://dev.mysql.com/doc/refman/5.7/en/server-error-reference.html#error_er_dup_entry
+                    if err.args[1] == 1062:
                         log.info(f'bunch containing job {(batch_id, jobs_args[0][1])} already inserted ({err})')
-                        return web.Response()
+                        raise web.Response()
```

Comment:
Can you explain to me why this is raise instead of return?

Reply:
@danking Can you respond?

Reply:
we're inside the insert function and I want to raise all the way up to the response handler